impl/src/main/java/io/kroxylicious/testing/kafka/testcontainers/TestcontainersKafkaCluster.java

```diff
@@ -144,6 +143,11 @@ public EndpointPair getControllerEndpoint(int brokerId) {
         }).collect(Collectors.toList());
     }
 
+    private void setDefaultKafkaImage(String kafkaVersion) {
+        DEFAULT_KAFKA_IMAGE = DockerImageName.parse("quay.io/ogunalp/kafka-native:" + kafkaVersion + "-snapshot");
```

Comment:
Let's extract "quay.io/ogunalp/kafka-native" as a constant.  We need some way to override that.  I *don't* think this needs to be exposed as an annotation. 

I think an environment variable or system property would be sufficient.

My use case are:
# I've just built kafka:next for the first time and it's in my own quay repo, I want to test it.
# I've hit this really knotty problem and I had to build a kafka image with extra instrumentation. I want an easy way to override it locally.

Reply:
done